<!DOCTYPE HTML>
<!--
	Escape Velocity by HTML5 UP
	html5up.net | @ajlkn
	Free for personal and commercial use under the CCA 3.0 license (html5up.net/license)
-->
<html>
	<head>
		<title>The Riesenfeld Group at UChicago---Research</title>
		<meta charset="utf-8" />
		<meta name="viewport" content="width=device-width, initial-scale=1, user-scalable=no" />
		<link rel="stylesheet" href="assets/css/main.css" />
		<link rel="icon" type="image/x-icon" href="favicon.ico">
	</head>
	<body class="homepage is-preload">
		<div id="page-wrapper">

			<!-- Header -->
				<section id="header" class="wrapper">

					<!-- Logo -->
						<div id="logo"></div>

					<!-- Nav -->
						<nav id="nav">
							<ul>
								<li><a href="index.html">Home</a></li>
								<li><a href="people.html">People</a></li>
								<li><a href="research.html">Research</a></li>
								<li><a href="publications.html">Publications</a></li>
								<li><a href="news.html">News</a></li>
							</ul>
						</nav>

				</section>

			<!-- Main -->
				<div id="main" class="wrapper style1">
					<div class="title">Research</div>
					<div class="container">

						<!-- Content -->
						
						<!-- Learning latent structure in 'omics data  -->
							<div id="content">
								<article class="box post">
									<header class="style1">
										<h2>Latent-factor multi-omic analysis of continuous immune-cell states</h2>
										<h3> Hope Anderson, Frank Gao, Hanna Hieromnimon, Joseph Sifakis </h3>
										<!-- <br class="mobile-hide" /> -->
									</header>
									<a href="#" class="image featured">
										<img src="images/website_continuous_states.png" alt="" />
									</a>
									<p class="style1">  
										Immune cells, such as innate lymphoid cells (ILCs), T helper cells, and macrophages, are traditionally 
										classified into discrete subsets based on their expression of specific transcription factors and effector cytokines. 
										The lens of single-cell RNA sequencing (scRNA-seq) has revealed a more complex spectrum of immune cell states and 
										functional plasticity across states. Standard, clustering-based analyses often have limited power to model such 
										phenomena. Instead, we represent the continuum of cell states using latent factorization techniques, which allow for 
										cells and genes to have a gradient of membership in different programs. These methods improve the power and 
										interpretability of downstream analyses in many of our applications. Thus, they play a role in several ongoing projects, 
										including the following:
										<ul style=“list-style-type:disc”>
											<li> Studying how the thymic development and T-cell receptor (TCR) signaling of human γδ T cells relate to their immune function </li>
											<li> Adapting methods to learn interpretable, interconnected gene programs </li>
											<li> Developing methods for RNA velocity (discussed below) </li>
											<li> Integrating histology and ‘omics data to discover better cancer biomarkers (discussed below) </li>
										</ul>
									</p>
								</article>
							</div>
					
						<!-- Dynamics -->
							<div id="content">
								<article class="box post">
									<header class="style1">
										<h2>System-wide transcriptional dynamics</h2>
										<h3> Frank Gao </h3>
										<!-- <br class="mobile-hide" /> -->
									</header>
									<a href="#" class="image featured">
										<img src="images/RNA_velocity_221111.png" alt="" />
									</a>
									<p class="style1">
										RNA velocity can reveal directions and flow patterns of transcriptional changes, even within a single 
										scRNA-seq time point, making it a potentially powerful tool. Yet, a lack of robustness and accuracy 
										has largely restricted applications to highly structured development and cell cycle contexts. To 
										empower RNA velocity for more general systems, including immune responses, we developed a new approach 
										that reflects our finding that parameter estimation depends critically on the input cells and genes. 
										While these are typically globally selected, we use probabilistic topic models, a type of latent 
										factorization, to infer the cells and genes specifically relevant for estimating process-specific 
										velocity parameters. We are now developing new methods that account for the stochastic nature of 
										lineage commitment, and we are investigating how to use RNA velocity to gain mechanistic insights into 
										cell fate decisions.
									</p>
									<p> </p>
								</article>
							</div>

							<!-- Integrating histology and 'omics data to discover better cancer biomarkers  -->
							<div id="content">
								<article class="box post">
									<header class="style1">
										<h2>Integrating histology and 'omics data to discover better cancer biomarkers</h2>
										<h3> Hanna Hieromnimon </h3>
										<!-- <br class="mobile-hide" /> -->
									</header>
									<a href="#" class="image featured">
										<img src="images/MRF_graphabs.png" alt="" />
									</a>
									<p class="style1">
										Improving cancer treatment relies on understanding the variables that drive key biological 
										processes in cancer. Studies have used molecular ‘omic data to predict diagnostic features, such as 
										cancer type and immune phenotype, but the cost and time of measuring gene expression impedes discovery 
										of clinical transcriptional biomarkers. Recent studies show that molecular information is encoded in 
										histology images, which are collected routinely. For example, deep learning models can be trained to 
										predict prognostic molecular features, such as genomic alterations, microsatellite instability, and 
										gene expression, directly from histology. We are motivated by the possibility of developing practical 
										clinical biomarkers by jointly leveraging genomics and histology using squamous cell carcinoma (SCC) 
										samples, encompassing lung, cervical, and head and neck cancer types. There is a critical unmet need 
										for comprehensive biomarkers that capture the complex biological processes underlying the cancer landscape. 
										Deep learning-based histology biomarkers offer a portable, cost effective, and accessible opportunity 
										to infer genomic information from histology anywhere in the world. Exploring the histology feature space 
										and connecting histologic features with biomarker prediction allows for the requisite explainability 
										for clinical integration.
										<br>
									</p>
									<p> </p>
								</article>
							</div>

							<!--Barrier immunity -->
							<div id="content">
								<article class="box post">
									<header class="style1">
										<h2>Complex cellular interactions in barrier tissue immunity</h2>
										<h3> Joseph Sifakis and Joey Federico </h3>
										<!-- <br class="mobile-hide" /> -->
									</header>
									<a href="#" class="image featured">
										<img src="images/mucosal_immunology_2.png" alt="" />
									</a>
									<p class="style1">
										There is growing recognition that the critical immune functions of barrier tissues, such as the 
										gut, lungs, and skin, depend not only on immune cells but also on complex interactions involving 
										microbes and other cell types. Dissecting these interactions is a major challenge, requiring a 
										combination of systematic transcriptomic analyses and targeted, hypothesis-driven experiments. 
										We have previously shown that in the gut, neuronal signaling is needed for mucus secretion 
										(accepted, Cell, 2022), and more generally, regional differences in epithelial cells prevent 
										adherent bacterial colonization and inflammation (Immunity, 2023). We are now studying how 
										gut-associated lymphoid tissues, such as Peyer’s patches, affect gut regionalization and barrier 
										immunity. Additionally, we are using RNA-seq to probe the mechanisms by which gut-resident 
										intraepithelial lymphocytes respond to cytokines implicated in autoimmunity.
										<br>
									</p>
									<p> </p>
								</article>
							</div>
						
							<!-- Manifolds  -->
							<div id="content">
								<article class="box post">
									<header class="style1">
										<h2>Learning Riemannian manifolds from point clouds</h2>
										<h3> Ryan Robinett </h3>
										<!-- <br class="mobile-hide" /> -->
									</header>
									<a href="#" class="image featured">
										<img src="images/MRF_graphabs.png" alt="" />
									</a>
									<p class="style1">
										Single-cell RNA-seq data highlights limitations of contemporary dimensionality reduction and 
										manifold learning techniques. Using simple test cases, studies have shown that key features of 
										the Riemannian metric and topological structure are sacrificed by commonly used methods such as 
										uniform manifold approximation and projection (UMAP), t-distributed stochastic neighborhood 
										embedding (tSNE), and Isomap. To preserve more structure, we have developed a data structure and 
										associated algorithms for constructing a finite atlas, a mathematical structure involving local 
										neighborhood coordinate charts, which can encode any topological manifold with compact closure. 
										Currently, we are testing the atlas construct in (non-biological) data settings with known Riemannian 
										manifold structure. Next, we plan to investigate which differential-geometric features of scRNA-seq 
										data can be learned with the atlas-graph structure and explore their biological significance.
									</p>
								</article>
							</div>
					</div>
				</div>

			<!-- Footer -->
				<section id="footer" class="wrapper">
					<div class="title">Contact Information</div>
					<div class="container">
						<div class="row">
							<div class="col-6 col-12-medium">

								<!-- Contact -->
									<section class="feature-list small">
										<div class="row">
											<div class="col-6 col-12-small">
												<section>
													<h3 class="icon solid fa-home">Physical Address</h3>
													<p>
														University of Chicago<br />
														900 E. 57th St.<br />
														Knapp Center for Biomedical Discovery, Rm. 9154<br />
														Chicago, IL 60637
													</p>
												</section>
											</div>
											<div class="col-6 col-12-small">
												<section>
													<h3 class="icon solid fa-comment">Social</h3>
													<p>
														<a href="#">@SJRiesenfeld</a><br />
													</p>
												</section>
											</div>
											<div class="col-6 col-12-small">
												<section>
													<h3 class="icon solid fa-envelope">Email</h3>
													<p>
														<a href="mailto: sriesenfeld@uchicago.edu">sriesenfeld@uchicago.edu</a>
													</p>
												</section>
											</div>
											<div class="col-6 col-12-small">
											</div>
										</div>
									</section>

							</div>
						</div>
						<div id="copyright">
							<ul>
								<li>Design: <a href="http://html5up.net">HTML5 UP</a></li>
							</ul>
						</div>
					</div>
				</section>

		</div>

		<!-- Scripts -->
			<script src="assets/js/jquery.min.js"></script>
			<script src="assets/js/jquery.dropotron.min.js"></script>
			<script src="assets/js/browser.min.js"></script>
			<script src="assets/js/breakpoints.min.js"></script>
			<script src="assets/js/util.js"></script>
			<script src="assets/js/main.js"></script>

	</body>
</html>
